azure_monitor/azfunc_sample/README.md

```diff
@@ -1,90 +1,78 @@
 ---
-page_type: sample
-languages:
-- python
-products:
-- azure
-description: "This sample contains a simple Azure Function to show how you can send correlated application log information as well as information of external dependencies to Azure Monitor (Application Insights)."
-urlFragment: azure-monitor-opencensus-python
+title: 'Sample: Configuring Function Apps with Python and OpenCensus'
+description: Create Azure Functions using Python and OpenCensus.
+author: givenscj
+ms.author: givenscj
+ms.devlang: python
+ms.topic: sample
+ms.date: 07/27/2022
+ms.custom: []
 ---
 
-# Correlated End-To-End Azure Function (Python) Instrumentation with Opencensus 
+# Sample: Configuring Function Apps with Python and OpenCensus
 
-## Overview
+This is an example of how to add the various settings Python-based Azure Function applications to send dependencies calls to Azure Monitor.  You will run the samples both locally and deploy the Azure Functions to an Azure App Service resource you deployed as part of the setup script.
 
-Azure Function (Python) already supports instrumentation using Opencensus. The following sample utilizes the OpenCensus Functions extension and enables additional common telemetry scenarios used by Azure Monitor and Functions users. Such scenarios include:
-- End-to-end view of a single function invocation (in Azure Monitor Application Insights)
-- Create an application topology view (application dependencies to support distributed tracing) using Azure Monitor Application Insight's Application Map (especially useful when working with distributed, scalable applications). 
+## 1 - Environment Setup
 
-### Example of a Correlated End-to-End Azure Function Invocation
+Follow the steps in the [setup documentation](/azure_monitor/readme.md).
 
-End-to-end transaction:
+## 2 - Utilize Function Apps with OpenCensus
 
-![AI Correlated end2end function invocation](./docs/assets/01-ai-correlated-invocation.PNG)
+- Start the Azure Storage Emulator.
+- Switch to Visual Studio Code.
+- Open a terminal window, run the following commands to start the Python functions locally:
 
-Associated application trace messages:
+```powershell
+cd .\azure_monitor\azfunc_sample
 
-![AI Application trace messages](./docs/assets/02-ai-correlated-invocation.PNG)
+python -m pip install -r requirements.txt
 
-Associated AI Application Map:
+func start
+```
```

Comment:
Can we have a quick explanation of what the function app is doing so that users will know what to expect in their telemetry?

Reply:
What does `MyHTTPFunction` do and `MyTimerFunction` do and how do they interact with each other and other components?

Reply:
Added details of the function apps.  I removed the timer function app as it is the same as the http one in terms of the core code.  Idea was to potentially have metrics with sampling for the timer, but we'll have to skip it.